### ISSUE TITLE
UltimMC can't be opened : mac problem.

### ISSUE THREAD

**Issue by GamerBoyTapan** [AfoninZ/UltimMC#191]
### Operating System

MacOS 12.0.1 (Monterey)

### Description of bug

It just says that and wont open. I tried dragging to applicatioon and still no work

### Steps to reproduce

idk

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by niklausthetitan** [comment]
Open the terminal, navigate to the directory the app is in and execute the following command:
`chmod +x UltimMC.app/Contents/MacOS/UltimMC`

**Comment by GamerBoyTapan** [comment]
Thanks but I think I have figgured the bug. There is no executable file in the app contents made for ultimMC to work. I have made executive file for the UltimMC which works. I will share it with you guys so people wont have trouble creating executive in future themselves.

https://drive.google.com/drive/folders/1DedX4cOO6PCY0GoQ2_wd7aNFJIgk8cFv?usp=sharing

Hope this works. Thank you

**Comment by GamerBoyTapan** [comment]
> Thanks but I think I have figgured the bug. There is no executable file in the app contents made for ultimMC to work. I have made executive file for the UltimMC which works. I will share it with you guys so people wont have trouble creating executive in future themselves.
> 
> https://drive.google.com/drive/folders/1DedX4cOO6PCY0GoQ2_wd7aNFJIgk8cFv?usp=sharing
> 
> Hope this works. Thank you

**Comment by GamerBoyTapan** [comment]
> Open the terminal, navigate to the directory the app is in and execute the following command: `chmod +x UltimMC.app/Contents/MacOS/UltimMC`

Thanks but I think I have figgured the bug. There is no executable file in the app contents made for ultimMC to work. I have made executive file for the UltimMC which works. I will share it with you guys so people wont have trouble creating executive in future themselves.

https://drive.google.com/drive/folders/1DedX4cOO6PCY0GoQ2_wd7aNFJIgk8cFv?usp=sharing

Hope this works. Thank you
